analyses/proteomic-data/proteomicsGSVA.R

```diff
@@ -0,0 +1,108 @@
+
+#load libraries
+library(tidyverse)
+library(readr)
+library(tibble)
+library(msigdbr)
+library(GSVA)
+
+
+#Magrittr pipe
+`%>%` <- dplyr::`%>%`
+#
+
+
+#set directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+#data directory
+data_dir <- paste(root_dir, "/data", sep="")
+#directory containing input files
+inputfile_dir <- "./output"
+#directory to store output
+output_dir <- "./GSEAoutput"
+#if the output directory does not exist, create it
+if (!dir.exists(output_dir)) {
+  dir.create(output_dir, recursive = TRUE)
+}
+#
+
+
+#read histology file to data frame
+histologyfile <- file.path(data_dir, "Hope-GBM-histologies-base.tsv")
+#HISTdata <- readr::read_tsv(histologyfile)
+HISTdata <- read.table(histologyfile, sep="\t", quote="\"", header=TRUE)
+#
+
+
+#WCP data input file
+quant_data_file <- file.path(inputfile_dir, "Hope_proteome_imputed_data_liftover.tsv")
+if (!file.exists(quant_data_file)){
+  stop("input quantification file does not exist")
+}
+#
```

Comment:
I think this file is output folder

Reply:
I anticipated that this would be confusing. I'm using the liftover datafile, which were written to the "output" directory when they were created. That directory is where I'm reading the file from, so I have the confusing situation of reading the data into the script from the output directory. I can change the directory names but I don't want to adversely impact others who might be working in this repo with the current directory structure. Happy to do what is best for everyone.